app/addons/documents/changes/components.react.jsx

```diff
@@ -11,34 +11,39 @@
 // the License.
 
 define([
+  'app',
+  'api',
   'react',
   'addons/documents/changes/actions',
-  'addons/documents/changes/stores'
-], function (React, Actions, Stores) {
+  'addons/documents/changes/stores',
+  'addons/fauxton/components.react',
+  'plugins/prettify'
+], function (app, FauxtonAPI, React, Actions, Stores, Components) {
 
+  var changesStore = Stores.changesStore;
   var ReactCSSTransitionGroup = React.addons.CSSTransitionGroup;
 
 
-  // the top-level component for the Changes Filter section. Handles hiding/showing
-  var ChangesHeader = React.createClass({
+  // the top-level component for the Changes Filter section. Handles hiding/showing of the filters form
+  var ChangesHeaderController = React.createClass({
     getInitialState: function () {
       return {
-        showTabContent: Stores.changesHeaderStore.isTabVisible()
+        showTabContent: changesStore.isTabVisible()
```

Comment:
This and line 37 could be made into a `getStoreState()` and then for `getInitialState` can just return `getStoreState` and for onchange you can do `this.setState(this.getStoreState());`